wear/SkeletonWear/SkeletonWear/GridExampleActivity.cs

```diff
@@ -66,22 +66,29 @@ public override Fragment GetFragment (int row, int col)
 			public override Android.Graphics.Drawables.Drawable GetBackgroundForPage (int row, int column)
 			{
 				Point pt = new Point (column, row);
-				Drawable drawable = mBackgrounds [pt];
-				if (drawable == null) {
-					var bm = Bitmap.CreateBitmap (200, 200, Bitmap.Config.Argb8888);
-					var c = new Canvas (bm);
-					var p = new Paint ();
-					// Clear previous image.
-					c.DrawRect (0, 0, 200, 200, p);
-					p.AntiAlias = true;
-					p.SetTypeface (Typeface.Default);
-					p.TextSize = 64;
-					p.Color = Color.LightGray;
-					p.TextAlign = Paint.Align.Center;
-					c.DrawText (column+ "-" + row, 100, 100, p);
-					drawable = new BitmapDrawable (owner.Resources, bm);
-					mBackgrounds.Add (pt, drawable);
-				}
```

Comment:
It's just visual studio format. I didn't use space here. It's weird how does it look here in github.

VisualStudio => http://prntscr.com/cnvq8b